packages/nice-grpc-client-middleware-devtools/LICENSE.md

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+
+Copyright (c) 2023 Sebastian Halder
```

Comment:
I'm not sure that we can license this to your name if we host it in this corporate repository. I may be wrong though.

If licensing to Deeplay does not sound right to you, you can host in in your own repo, and we will make a link to it in our docs.